spec/lib/tasks/annotate_models_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../../spec_helper'
+
+describe 'Annotate annotate_models rake task and Annotate.set_defaults' do # rubocop:disable RSpec/DescribeClass
```

Comment:
@ctran : My apologies for bypassing a `rubocop` rule here, but there is no class to describe Rake tasks, so that would justify the exception.

Also, I do not understand why `rubocop` is not complaining about `spec/lib/tasks/annotate_models_migrate_spec.rb`, since it also has a free string test root describe block.

Reply:
It was excluded: https://github.com/ctran/annotate_models/blob/develop/.rubocop_todo.yml